src/UserRepository.js

```diff
@@ -1,5 +1,21 @@
 class UserRepository {
+  constructor(data) {
+    this.data = data;
+  }
 
+  renderUserData(userId) {
+    const userData = this.data.find((user) => user.id === userId);
+    return userData;
+  }
+
+  calculateAvgUserStepGoal() {
+    const totalUserSteps = this.data.reduce((accumulator, user) => { 
+      return accumulator += user.dailyStepGoal; 
+    }, 0);
+
+    return parseInt(totalUserSteps/this.data.length);
+  }
```

Comment:
Every user's dailyStepGoal will be added to the accumulator (lines 12-14). Then we'll return the average user's dailyStepGoal by taking the result of the accumulator (totalUserSteps) and dividing it by the number of users that we have (this.data.length). parseInt will round down to the nearest whole number.

Reply:
I think we could refactor parseInt to Math.floor() or Math.round() since we're taking in a number.

Reply:
I'm not really familiar with math.floor or math.round. But I love working with a math major!